src/game/component/PhysicsComponent.h

```diff
@@ -35,4 +35,6 @@ class PhysicsComponent : public IComponent
 	glm::vec3 m_Velocity;
 	float m_ImpulseWait;
 	MovableComponent* m_pMover;
+
+	Sound *sound = ResourceManager::returnSound();
```

Comment:
Physics code shouldn't have anything to do with sounds. The sound code here and in PhysicsComponent.cpp should be handled in Action_Jump